atd-vze/src/views/Crashes/Crash.js

```diff
@@ -271,13 +273,19 @@ function Crash(props) {
       )}
       <Row>
         <Col>
-          <CrashCollapses
-            data={crashData.crashes_by_pk}
-            refetch={crashRefetch}
-            loading={loading}
-            error={error}
-            props={props}
-          />
+          <Card>
```

Comment:
Super small detail but this card wrapper is giving this section an extra wide border and some white background that is revealed by the border radius where contained cards meet.

Reply:
🙏 thank you — i just cleaned this up in my the custom ID branch: https://github.com/cityofaustin/atd-vz-data/pull/1465/commits/a4de86929be7e794d46d5fc7313bf43978dfba23